### DISCUSSION TITLE
Exception in thread "main" java.lang.NoClassDefFoundError: com/sun/xml/bind/v2/model/annotation/AnnotationReader

### DISCUSSION THREAD

**Discussion by jordanpadams** [NASA-PDS/transform#66] [Q&A] [answered]
i run this tool and then something goes wrong:

![283696667-1f901a62-8073-4d04-be7e-cedba2eeaf4c](https://github.com/NASA-PDS/transform/assets/33492486/65fc0ad4-c178-4097-8998-3afd027871ea)

**Comment by pds-ops** [comment, marked as answer]
This usually indicates that your are using the wrong version of Java when executing the software. Per the system requirements, Transform Tool requires Java 11-17 (and probably works for later versions as well).

Be sure to not only ensure the new version of Java is in your PATH, but also that your `JAVA_HOME` is not set to an older version:

```
$ which java
```

```
$ echo $JAVA_HOME
```
